lib/api/controllers/collectionController.js

```diff
@@ -62,7 +75,7 @@ class CollectionController {
     assertHasIndexAndCollection(request);
 
     return this.internalEngine.get('validations', `${request.input.resource.index}#${request.input.resource.collection}`)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 121 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)